testing/src/main/java/org/oppia/android/testing/platformparameter/TestPlatformParameterModule.kt

```diff
@@ -204,6 +259,7 @@ class TestPlatformParameterModule {
     private var enablePerformanceMetricsCollection =
       ENABLE_PERFORMANCE_METRICS_COLLECTION_DEFAULT_VALUE
     private var enableSpotlightUi = true
+    private var enableAppAndOsDeprecation = ENABLE_APP_AND_OS_DEPRECATION_DEFAULT_VALUE
```

Comment:
We should also be able to restore the default value of this in the ` reset()` function below

Reply:
I wasn't aware of the `reset()` function. I have added the ability to reset the `enableAppDeprecation` variable on the `reset()` function.

Reply:
You can use this in your test setup to reset the platform values before each test run.